rwengine/src/audio/SfxParameters.hpp

```diff
@@ -0,0 +1,133 @@
+    {78, 2857/*-2871*/, -1},        /// male pain soft
+    {79, 2857/*-2871*/, -1},        /// male pain loud
+    {80, 2290/*-2300*/, -1},        /// female pain soft
+    {81, 2290/*-2300*/, -1},        /// female pain loud
+    {82, 161/*-162*/, -1},          /// item pickup
+    {83, 161/*-162*/, -1},          /// item pickup
+    {92, 147, 40},                  /// gate start
+    {93, 147, 40},                  /// gate stop
+    {94, 337, -1},                  /// checkpoint
+    {97, 336, -1},                  /// countdown timer 3
+    {98, 336, -1},                  /// countdown timer 2
+    {99, 336, -1},                  /// countdown timer 1
+    {100, 337, -1},                 /// countdown finish
+    {106/*-108*/, 176/*-178*/, 50}, /// bullet hit ground
+    {110/*-111*/, 389, 80},         /// silent, replace the SFX or edit the memory to a different SFX to make this useful
+    {112, 283, 80},                 /// payphone
+    {114/*-115*/, 151, 60},         /// glass break
+    {116, 150, 60},                 /// glass damage
+    {117, 152/*-155*/, 55},         /// glass shards
+    {118, 327, 60},                 /// boxes destroyed
+    {119, 328, 60},                 /// boxes destroyed
+    {120, 140/*-144*/, 60},         /// car collision
+    {121, 29, 60},                  /// tire collision
+    {122, 167/*-168*/, 20},         /// gun shell drop, dependent on what collision the player is currently standing on
+    {123, 168, 20},                 /// gun shell drop soft
+    /// for opcodes 018d
+    {4, 390, 30},
+    {5, 390, 80},                   /// Hepburn Heights southwest tower
+    {6, 391, 30},
+    {7, 391, 80},                   /// Hepburn Heights north tower
+    {8, 392, 30},
+    {9, 392, 80},
+    {10, 393, 30},
+    {11, 393, 80},
+    {12, 394, 30},
+    {13, 394, 80},
+    {14, 395, 30},
+    {15, 395, 80},
+    {16, 396, 30},
+    {17, 396, 80},
+    {18, 397, 30},
+    {19, 397, 80},
+    {20, 398, 30},
+    {21, 398, 80},
+    {22, 399, 30},
+    {23, 399, 80},
+    {24, 390, 30},
+    {25, 390, 80},
+    {26, 391, 30},
+    {27, 391, 80},
+    {28, 392, 30},
+    {29, 392, 80},
+    {30, 403, 30},                  /// Meeouch Sex Kitten Club
+    {31, 403, 80},
+    {32, 404, 30},                  /// Sex Club Seven
+    {33, 404, 80},
+    {34, 405, 30},
+    {35, 405, 30},
+    {36, 407/*-408*/, 30},          /// 407 plays continuously while 408 plays intermittently
+    {37, 407/*-408*/, 30},          /// Liberty City Sawmills/Bitch'N' Dog Food
+    {38, 409, 30},
+    {39, 409, 80},
+    {40, 410/*-411*/, 30},          /// Both plays continuously
+    {41, 410/*-411*/, 30},          /// Mr. Wong's Laundrette
+    {42, 412, 30},                  /// Roast Peking Duck
+    {43, 412, 80},
+    {44, 413, 30},                  /// Cipriani's Ristorante
+    {45, 413, 80},
+    {46, 414, 30},
+    {47, 414, 30},
+    {48, 415, 30},                  ///	Marco's Bistro (no sound)
```

Comment:
The comments do not have consistent layout + should not contain tabs.

Reply:
I forgot to reformat it. (shame)